src/analyze/analyze-security.c

```diff
@@ -1824,6 +1871,44 @@ static int property_read_ip_address_allow(
         return sd_bus_message_exit_container(m);
 }
 
+static int property_read_ip_filters(
+                sd_bus *bus,
+                const char *member,
+                sd_bus_message *m,
+                sd_bus_error *error,
+                void *userdata) {
+
+        struct security_info *info = userdata;
+        int r;
+
+        assert(bus);
+        assert(member);
+        assert(m);
+
+        r = sd_bus_message_enter_container(m, 'a', "s");
+        if (r < 0)
+                return r;
+
+        for (;;) {
+                const char *path;
+
+                r = sd_bus_message_read(m, "i", &path);
```

Comment:
this can't be right. you are reading an integer here. Needs to be a string

Reply:
Good catch, thanks.